.github/ISSUE_TEMPLATE/newrepo.yml

```diff
@@ -0,0 +1,166 @@
+          - repo is public with same group/permissions as the main Egeria repo
+          - If using maven/gradle, snapshots will be published to oss.sonatype.org and releases to maven central
+          - If using containers, they will be published to quay.io & docker.io & built for arm64 + amd64 with same release versions as other artifacts
+          - If using maven, versioning should follow 1.1 / 1.1-SNAPSHOT convention. For javascript etc use semver, ie 1.1.1 / 1.1.1-rc.0
+          - Follows egeria branching-strategy ie branch just before release
+          - Security policy aligned with egeria - common mailing list for reporting vulnarabilities
+      placeholder: |
+        Yes - all true
+    validations:
+      required: true
+  - type: textarea
+    id: extra
+    attributes:
+      label: Additional Information
+      description: |
+        Any other information?
+      placeholder: |
+        add any other information here
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+        label: Work Plan
+        description: | 
+          DO NOT FILL THIS IN NOW
+          
+          This is a checklist to guide the work needed to complete getting the repository and code into an appropriate state for ongoing development. 
+                
+          Where these items may not be undertaken soon after creation, an issue should be opened to track.
+        value: |
+            Before creating the repo
+            - [ ] review overall request & get clarifications
+            - [ ] get approval on developer/TSC call from maintainers
+          
+            Creating the repo
+            - [ ] Create the repo under the odpi organization (default gitignore, license, readme)
+            - [ ] Setup branch protection rules
+            - [ ] Set pull request options (allow merge, squash, rebase, suggest updating, allow automerge, do not delete head)
+            - [ ] Update security settings in repo (policy, advisories, alerts)
+            - [ ] set up permissions
+          
+            First steps
+            - [ ] Initial code-drop (author)
+          
+            Getting CI/CD started & refining settings
+            - [ ] Add initial build script for PR (including gradle wrapper if required)
+            - [ ] Add initial build script for merge
+          
+            Further Refinement
+            - [ ] Add link in egeria docs to new repo describing purpose etc       
+            - [ ] Add required credentials for publishing to container repos, maven central etc
+            - [ ] Add artifact signing if needed
+            - [ ] Add dependabot config        
+            - [ ] Add CodeQL 
+            - [ ] Add to LFXSecurity
+            - [ ] Add to LFAnalytics
+            - [ ] Add check for stale defects
+            - [ ] Add standard issue tags
+          
+            Release
+            - [ ] Add release pipeline
```

Comment:
add that the release process should be documented - with the link to the standard place.

Reply:
Adding to https://github.com/odpi/egeria/issues/6298